src/components/Calculator.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Input from './Input';
+
+function CalculatorDesign() {
+  return (
+    <div className="parent">
+      <Input />
+      <button className="child light-grey" type="button">AC </button>
+      <button className="child light-grey" type="button">+/- </button>
+      <button className="child light-grey" type="button">% </button>
+      <button className="child orange" type="button">/</button>
+      <button className="child light-grey" type="button">7</button>
+      <button className="child light-grey" type="button">8</button>
+      <button className="child light-grey" type="button">9</button>
+      <button className="child orange" type="button">x</button>
+      <button className="child light-grey" type="button">4</button>
+      <button className="child light-grey" type="button">5</button>
+      <button className="child light-grey" type="button">6</button>
+      <button className="child orange" type="button">-</button>
+      <button className="child light-grey" type="button">1</button>
+      <button className="child light-grey" type="button">2</button>
+      <button className="child light-grey" type="button">3</button>
+      <button className="child orange" type="button">+</button>
+      <button className="child zero light-grey" type="button">0</button>
+      <button className="child light-grey" type="button">.</button>
+      <button className="child orange" type="button">=</button>
+    </div>
+  );
```

Comment:
- [ ] Excellent work on the Calculator component 👏. However, as stated in the requirements, at least one children component is required. The goal is for you to learn how to pass state from one component to another using props, which is a useful skill to have as a React developer. :ok_hand: 😊
So I'd recommend moving these features to a different component and importing it here as a child component.

- [ ] Last but not least, in order to adhere to React best practices, I would advise you to ensure that the name of the component is the same as the name of the file. The component in this case should be called **Calculator**.  🤗


#### Requirement for having at least one child component :point_down: 

![Screenshot from 2023-08-12 15-41-59](https://github.com/MargaretTechworld/math-magicians/assets/89228743/a7a651e9-744d-4615-a107-fb53f143d300)